templates/riscv/qemu-sifive-u/mods.config

```diff
@@ -21,5 +22,9 @@ configuration conf {
 	include embox.compat.libc.math_builtins
 	include embox.kernel.spinlock(spin_debug = false)
 
+	include embox.kernel.task.resource.errno
+
 	include embox.test.stdlib.setjmp_test
+	include embox.test.kernel.timer_test
+	include embox.test.hal.context_switch_test(stack_sz=0x40)
```

Comment:
Maybe you should switch on more multi-threading features?

Reply:
I think I'll do it later, after implementing vfork.